tools/WixBuild.vcxproj.props

```diff
@@ -52,6 +53,8 @@
   <PropertyGroup>
     <ResourceOnlyDll Condition=" '$(ResourceOnlyDll)'=='' ">false</ResourceOnlyDll>
     <GenerateManifest Condition="$(ResourceOnlyDll)">false</GenerateManifest>
+    <WindowsSDKDesktopARMSupport Condition=" '$(Platform)'=='arm' and $(VS2017Available) ">true</WindowsSDKDesktopARMSupport>
```

Comment:
ARM .libs are still included in the Windows 10 SDK if you don't select ARM build tools